sql/core/src/main/scala/org/apache/spark/sql/streaming/StreamingQueryManager.scala

```diff
@@ -219,10 +219,11 @@ class StreamingQueryManager private[sql] (sparkSession: SparkSession) {
       }
 
       // If offsets have already been created, we trying to resume a query.
-      if (!recoverFromCheckpointLocation) {
+      if (recoverFromCheckpointLocation) {
```

Comment:
If I correctly interpret [this comment](https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/streaming/StreamingQueryManager.scala#L181-L183), an exception will be thrown when `recoverFromCheckpointLocation` == `false` and `fs.exists(checkpointPath)` == `true` are satisfied.
Is [the comment](https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/streaming/StreamingQueryManager.scala#L181-L183) wrong if this line is correct?